hushline/routes.py

```diff
@@ -518,10 +526,15 @@ def logout() -> Response:
     @app.route("/directory")
     def directory() -> Response | str:
         logged_in = "user_id" in session
+        # Provide a default intro text if DIRECTORY_INTRO is not set
```

Comment:
nit: unnecessary comment. there's a bunch of these actually and they should be yanked because they simply state exactly what the line is doing whereas comments should be to explain business logic or clarify code that cannot be made less convoluted.

Reply:
✅